### PR TITLE
Implement checkout command

### PR DESCRIPTION
**Pull request by tomer-stripe**
### Reviewers
r? @ob-stripe 
cc @stripe/dev-platform @cjavilla-stripe 

 ### Summary
This adds a command to help users test checkout be precreating the checkout session or letting them pipe it in:

```
$ stripe checkout run
```

or

```
$ echo {id: cs_1234foo} | stripe checkout run
```

### CODE REVIEW

**Comment by tomer-stripe** [comment]
ptal @ob-stripe 

Minus the html files this is ready!

File: pkg/checkout/session.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+func createCheckoutSession(cfg *config.Config) (string, error) {
+	secretKey, err := cfg.Profile.GetAPIKey(false)
+	if err != nil {
+		return "", err
+	}
+
+	data := requests.RequestParameters{}
+	data.AppendData([]string{
+		`success_url=http://localhost:4242/success`,
+		`cancel_url=http://localhost:4242/cancel`,
```
Comment (ob-stripe) [inline pkg/checkout/session.go]: Shouldn't these two URLs be parameterized with the port number?

Comment (tomer-stripe) [reply pkg/checkout/session.go]: Oh, good catch! Fixing

**Comment by tomer-stripe** [comment]
Closing this -- based on feedback it sounds like there wont be much interest for users.

**Pull request closed without merging**